### PR TITLE
MER_TX_TB_Report

### PR DESCRIPTION
**Pull request by dbaluku**
https://app.asana.com/0/1133167201254915/1200162047658942/f

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ *  PMTCT STAT Report
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java:26]: @dbaluku Change title to TX_TB

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.ugandaemrreports.library.*;
+import org.openmrs.module.ugandaemrreports.metadata.HIVMetadata;
+import org.openmrs.module.ugandaemrreports.reports.SetupTxNewReport;
+import org.openmrs.module.ugandaemrreports.reports.UgandaEMRDataExportManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.awt.*;
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java:20]: @dbaluku Why do you need the awt? Remove Wildcard imports

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+        addGender(dsd,"11","previously on ART start TB treatment screen negative below 15 years ", df.getPatientsInAll(havingArtStartDateBeforeQuarter,screenedNegative,below15Years));
+        addGender(dsd,"12","previously on ART start TB treatment screen negative above 15 years ", df.getPatientsInAll(havingArtStartDateBeforeQuarter,screenedNegative,above15Years));
+
+
+
+        return rd;
+    }
+
+    public CohortDefinitionDimension getGender(){
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java:162]: @dbaluku Why not put this in the CohortDefinitionDimension library to be used in other reports

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java (hunk 1 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java]: Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java (hunk 2 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java]: Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java (hunk 3 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupMER_TX_TBReport.java:165]: This is mapping report parameters, this cant be general

**Pull request merged**